datamodel/src/main/java/com/hartwig/serve/datamodel/EvidenceLevelDetails.java

```diff
@@ -0,0 +1,26 @@
+package com.hartwig.serve.datamodel;
+
+public enum EvidenceLevelDetails {
+    PRECLINICAL("Preclinical"),
+    CASE_REPORTS_SERIES("Case report series"),
+    CLINICAL_STUDY("Clinical study"),
+    FDA_APPROVED("FDA approved"),
+    GUIDELINE("Guideline"),
+    FDA_CONTRAINDICATED("FDA contraindicated"),
+    UNKNOWN("Unknown");
+
+    public String description;
+
+    EvidenceLevelDetails(String description) {
+        this.description = description;
+    }
+
+    public String getDescription() {
+        return description;
+    }
+
+    public EvidenceLevelDetails evidenceLevelDetailsCreator(String description) {
+        this.description = description;
+        return this;
+    }
```

Comment:
The description should be a private final variable (annotated with NotNull), and not settable. See also other comment